src/components/Uploader/style.css

```diff
@@ -43,9 +42,11 @@
   cursor: pointer;
   @media (--large) {
     min-height: 20rem;
+    margin: 1rem;
+    min-height: 20rem;
+    height: 264px;
```

Comment:
Could you check that the new UI matches the design? In this case the dropzone is too large because of the `min-height` (which is also repeated)

Reply:
Good spot, I have to double check all the new changes that Daniel have made to the design after I have already implementing the original design. Will update it

Reply:
There is a ticket for that in Jira https://jira.onfido.co.uk/browse/CX-1217
We haven't started the debate yet. We can use both rem and px at the same time, it's not a problem, as long as we do it properly so that the UI looks good when a user changes the front size in their browser settings. I wonder what's the percentage of users that change the font size in their browser settings, and if it's big enough maybe we should tackle this issue as part of the accessibility epic, whenever we will work on that.